beets/library.py

```diff
@@ -1145,6 +1145,9 @@ def _getters(cls):
     def items(self):
         """Return an iterable over the items associated with this
         album.
+
+        This method predates the :meth:`LibModel.items` method inherited
+        from :meth:`beets.dbcore.Model.items`.
```

Comment:
Maybe make it even clearer what this is about (at the risk of becoming overly verbose)? Along the lines of

```suggestion
        This method conflicts with `.items()` in Python's mapping protocol, 
        which is implemented by :meth:`LibModel.items` method (inherited
        from :meth:`beets.dbcore.Model.items`).
        Since :meth:`Album.items` predates these methods, and is likely to be used
        by plugins, we keep this interface as-is.
```

Reply:
Yes that's a good idea. I think the bit about the mapping protocol might be overly verbose; maybe just have something like this?

```
        This method conflicts with :meth:`LibModel.items`, which is inherited
        from :meth:`beets.dbcore.Model.items`.
        Since :meth:`Album.items` predates these methods, and is likely to be
        used by plugins, we keep this interface as-is.
```

Reply:
I don't have a clear preference here: It's up to you :)